### PR TITLE
peng/527-onboarding

### LINKED ISSUE #527: Onboarding wizard
> Thank you for filing an issue!

### Version

0.4.3

### Description

The domain of crypto is very new and strange to users. To archive a first class UX we want to have an onboarding wizard explaining most of the steps and functionalities to the user

### PR DESCRIPTION
**Pull request by nylira**
### Issue
Closes #527 

- [x] create modals for onboarding steps
- [x] create images for onboarding steps
- [x] persist onboarding finished status in localstorage
- [x] integrate onboarding images into onboarding modal
- [x] unit tests

### Screenshots

Here are the five steps of onboarding (WIP). The final images are in a comment further down the page.

screen shot 2018-04-30 at 2 05 31 pm

### CODE REVIEW

**Comment by nylira** [comment]
Here are the final screenshots. Going to work on fixing tests now.

screen shot 2018-05-01 at 2 04 30 pm
screen shot 2018-05-01 at 2 04 53 pm
screen shot 2018-05-01 at 2 05 04 pm
screen shot 2018-05-01 at 2 06 41 pm
screen shot 2018-05-01 at 2 06 51 pm

**Comment by nylira** [comment]
This is ready for review. If the changes are small, please make the edit yourself and then merge the PR. It would be great to get this in for the Tuesday meeting later.

**Comment by jolesbi** [comment]
so nice peng. well done!

**Comment by NodeGuy** [comment]
Beautiful images!

**Comment by mappum** [comment]
Great job @nylira !

File: app/src/renderer/components/common/NiOnboarding.vue [hunk 1]
```diff
@@ -44,7 +44,7 @@ export default {
       "You can send and receive Cosmos tokens from anyone around the world.",
       "You can stake your Atoms to Cosmos Validators to earn even more Atoms.",
       "Through governance, you can vote on the future of the Cosmos Network.",
-      "That's everything. Start using Voyager to explore the Cosmos Network!"
+      "Start using Voyager to explore the Cosmos Network!"
```
Comment (NodeGuy) [inline app/src/renderer/components/common/NiOnboarding.vue]: 👍

**Pull request merged**